R/rbindFill.R

```diff
@@ -58,7 +58,7 @@ rbindFill <- function(...) {
     allnms <- unique(names(allcl))
     allcl <- allcl[allnms]
 
-    for (i in seq(along=l)) {
+    for (i in seq(along.with = l)) {
```

Comment:
We use `seq_along` everywhere else.

```suggestion
    for (i in seq_along(l)) {
```